components/board/BestArticle.tsx

```diff
@@ -49,27 +51,46 @@ function BestArticle({ article }: BestArticleProps) {
               <p className={styles["like-count"]}>{article.likeCount}</p>
             </div>
           </div>
-          <p className={styles.date}>
-            {moment(article.createdAt).format("YYYY.MM.DD")}
-          </p>
+          <p className={styles.date}>{getDate(article.createdAt)}</p>
         </div>
       </article>
     </Link>
   );
 }
 
 // NOTE - 베스트 게시글 리스트
-interface BestArticleListProps {
-  bestArticleList: Article[];
-}
 
-export default function BestArticleList({
-  bestArticleList,
-}: BestArticleListProps) {
+export default function BestArticleList() {
+  const { isDesktop, isMobile, isTablet, bestPageSizeCount } = useDeviceSize();
+  const [bestArticleList, setBestArticleList] = useState<Article[]>([]);
+  const [bestPageSize, setBestPageSize] = useState(bestPageSizeCount);
```

Comment:
@ssseeo0 
얘도 마찬가지구요 ㅎㅎ

```suggestion
  const [pageSize, setPageSize] = useState(bestPageSizeCount);
```